src/lib.rs

```diff
@@ -170,14 +170,17 @@ mod test {
             };
             let variants = s
                 .variants()
-                .map(|(m, v, _)| (m.into_iter().collect::<BTreeSet<_>>(), v))
+                .map(|(m, v, _)| {
+                    (m.into_iter().map(|m| m.as_str()).collect::<BTreeSet<_>>(), v)
```

Comment:
```suggestion
                .map(|(ms, v, _)| {
                    (ms.into_iter().map(|m| m.as_str()).collect::<BTreeSet<_>>(), v)
```

Or something else to make it clear that this is a modifier _set_.